tests/integration/parse.spec.ts

```diff
@@ -0,0 +1,115 @@
+// tslint:disable-next-line:no-require-imports no-implicit-dependencies
+import express = require('express');
+
+import {
+    Controller,
+    ApiController,
+    Dino,
+    HttpGet,
+    Parse,
+    toInteger,
+    toNumber,
+    toBoolean,
+    request
+} from '../index';
+
+@Controller('/testcontroller')
+class TestControllerFake extends ApiController {
+    @HttpGet('/toBoolean/:bool')
+    toBoolean(@Parse(toBoolean) bool: boolean): any {
+        return { bool };
```

Comment:
Can we have this is as `{ data: bool }` so that it will be consistent across all test cases. Instead of having different types always.